src/dbus/server/introspect.xml

```diff
@@ -337,6 +356,26 @@
     <property name="ExternalRoutes" type="((ayy)qybb)" access="read">
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
     </property>
+
+    <signal name="CommissionerState">
```

Comment:
I think we should define another object for commissioner. For example: /io/openthread/commissioner
Thus we can keep the same or similar dbus API with external commissioner.